lib/helper/WebDriver.js

```diff
@@ -906,15 +906,24 @@ class WebDriver extends Helper {
   }
 
   /**
-   * {{> grabTextFrom }}
+   * {{> grabTextFromAll }}
    *
    */
-  async grabTextFrom(locator) {
+  async grabTextFromAll(locator) {
     const res = await this._locate(locator, true);
     assertElementExists(res, locator);
```

Comment:
Please move this line to `grabTextFrom`, to ensure that 1 result will be available

Reply:
Done